more_itertools/more.py

```diff
@@ -2401,3 +2401,35 @@ def time_limited(limit_seconds, iterable):
         if monotonic() - start_time > limit_seconds:
             break
         yield item
+
+
+def only(iterable, default=None, too_long=None):
+    """If *iterable* has only one item, return it.
+    If it has zero items, return *default*.
+    If it has more than item, raise the exception given by *too_short*,
```

Comment:
"... more than *one* item ..."

`too_short` looks like it should  be `too_long`.